src/main/java/org/multiplex/domain/Reservation.java

```diff
@@ -0,0 +1,43 @@
+package org.multiplex.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Value;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import java.time.OffsetDateTime;
+import java.util.Set;
+
+@Entity
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+class Reservation {
```

Comment:
Brakuje mi całkowitej sumy do zapłaty, jest ona wyliczona tylko przy rezerwacji, czy widzisz potencjalne zagrożenia?

Reply:
Rzeczywiście, wyliczona suma powinna być zapisana w bazie. Może być ona potrzebna np. przy rezygnacji z rezerwacji albo reklamacji, kino musi wiedzieć ile powinno zwrócić pieniędzy, a nie można zakładać, że cenny biletów zawsze będą takie same. Dodatkowo dana ta może być potrzebna do celów raportowych albo analitycznych np. do sprawdzenia czy na koniec miesiąca przychody były takie jakby to wynikało z liczby sprzedanych biletów.